src/burnchains/burnchain.rs

```diff
@@ -365,6 +365,35 @@ impl Burnchain {
         })
     }
 
+    /// the expected sunset burn is:
+    ///   total_commit * (progress through sunset phase) / (sunset phase duration)
+    pub fn expected_sunset_burn(&self, burn_height: u64, total_commit: u64) -> u64 {
+        if burn_height < self.pox_constants.sunset_start
+            || burn_height >= self.pox_constants.sunset_end
+        {
+            return 0;
```

Comment:
Shouldn't this be `total_commit` if `burn_height >= self.pox_constants.sunset_end`?

Reply:
No -- the sunset_burn isn't ever used to determine the total commitment amount -- just the `burn_fee` field.